persistence/docs/README.md

```diff
@@ -90,67 +90,107 @@ persistence         # Directly contains the persistence module interface for eac
 
 ### Makefile Helpers
 
-If you run `make` from the root of the `pocket` repo, there will be several targets prefixed with `db_` that can help with design & development of the infrastructure associated with this module
+Running `make` from the root of the `pocket` repo will show several targets prefixed with `db_` to help with the design, development & debugging of this module.
 
-We only explain a subset of these in the list below.
+A subset of these are explained below.
 
-#### Admin View - db_admin
+#### CLI Access - db_cli_node
+
+Open up a CLI to the postgres database via:
+
+```bash
+make db_cli # Connect to the DB before connecting to a specific schema
+# OR
+psqlSchema=node3 make db_cli_node # Immediately connect to the schema of node3
+```
 
-When you run `db_admin`, the following will be echoed to your screen
+Then you can make use of the following SQL commands and explore:
 
+```sql
+SELECT schema_name FROM information_schema.schemata; // View all of the schemas available
+SET search_path=node1; // Set the search path to a schema `node1`
+/dt // View all available tables
 ```
+
+#### Admin View - db_admin
+
+<!--
+# IMPROVE: Bind the login credentials to the postgres container so its automatically logged in and debug why the docker-compose configurations do not automatically prepare the server
+-->
+
+Running `make db_admin` should echo the following to your terminal:
```

Comment:
May be worth saying "_requires `make compose_and_watch` to have been run for nodes to be initialised and for their schemas and tables to be present in any of the `make db_...` commands. `make db_start` does not do this in place of `make compose_and_watch`_"

This could be at the start of the `make db_...` section? Or FAQ

Reply:
Built on top of your changes and pushed